src/server/handlers/consents/{ID}.ts

```diff
@@ -119,7 +120,7 @@ export async function put (_context: Context, request: Request, h: ResponseToolk
   const destinationParticipantId = request.headers[Enum.Http.Headers.FSPIOP.SOURCE]
 
   // Note: not awaiting promise here
-  validateAndUpdateConsent(id, updateConsentRequest, destinationParticipantId)
+  thisModule.validateAndUpdateConsent(id, updateConsentRequest, destinationParticipantId)
```

Comment:
I'm not sure I understand the need for this here - is that just to get the mocks to work?

Reply:
Yes, I tested without doing this and the original function is still called.
This was what I referenced: https://stackoverflow.com/questions/45111198/how-to-mock-functions-in-the-same-module-using-jest/45288360